server/ai_mediaserver.go

```diff
@@ -809,6 +809,7 @@ func (ls *LivepeerServer) CreateWhip(server *media.WHIPServer) http.Handler {
 			whepURL = "http://localhost:8889/" // default mediamtx output
 		}
 		whepURL = whepURL + streamName + "-out/whep"
+		conn := server.CreateWHIP(ctx, ssr, whepURL, w, r)
```

Comment:
Afaict it looks ok to move this line up/earlier but maybe worth @j0sh confirming